src/net/sourceforge/kolmafia/textui/command/MayamCommand.java

```diff
@@ -0,0 +1,103 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.sourceforge.kolmafia.KoLConstants.MafiaState;
+import net.sourceforge.kolmafia.KoLmafia;
+import net.sourceforge.kolmafia.RequestThread;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.GenericRequest;
+import net.sourceforge.kolmafia.session.InventoryManager;
+
+public class MayamCommand extends AbstractCommand {
+  private static final List<List<String>> SYMBOL_POSITIONS =
+      List.of(
+          List.of("yam", "clock", "explosion"),
+          List.of("yam", "eyepatch", "cheese", "wall"),
+          List.of("yam", "lightning", "bottle", "board", "meat"),
+          List.of("yam", "sword", "vessel", "fur", "chair", "eye"));
+
+  public MayamCommand() {
+    var symbols =
+        SYMBOL_POSITIONS.stream().flatMap(List::stream).distinct().collect(Collectors.joining("|"));
+    this.usage = "[" + symbols + "] - list symbols top to bottom (largest ring to smallest)";
+  }
+
+  private boolean lacksCalendar() {
+    if (!InventoryManager.equippedOrInInventory(ItemPool.MAYAM_CALENDAR)) {
+      KoLmafia.updateDisplay(MafiaState.ERROR, "You need a Mayam Calendar");
+      return true;
+    }
+    return false;
+  }
+
+  @Override
+  public void run(final String cmd, String parameters) {
+    String[] symbols = parameters.split(" ");
```

Comment:
should verify that there are exactly four symbols passed in, and fail otherwise.

If less, it'll use the default ring symbols. If more, it'll crash.